Small formatting issue
Run dart format --set-exit-if-changed .
Formatted packages/smooth_app/lib/database/dao_hive_product.dart
Formatted 3[8](https://github.com/openfoodfacts/smooth-app/actions/runs/6922206431/job/18828662636?pr=4821#step:7:9)1 files (1 changed) in 1.55 seconds.
Error: Process completed with exit code 1.

--- a/packages/smooth_app/lib/database/dao_hive_product.dart
+++ b/packages/smooth_app/lib/database/dao_hive_product.dart
@@ -34,8 +34,7 @@ class DaoHiveProduct extends AbstractDao implements DaoProductMigrationSource {
   Future<void> init() async => Hive.openLazyBox<Product>(_hiveBoxName);
 
   @override
-  void registerAdapter() =>
-    Hive.registerAdapter(_ProductAdapter());
+  void registerAdapter() => Hive.registerAdapter(_ProductAdapter());
 
   LazyBox<Product> _getBox() => Hive.lazyBox<Product>(_hiveBoxName);
 

